apps/dashboard/app/models/batch_connect/session.rb

```diff
@@ -73,7 +73,7 @@ def app
     end
     
     # How many days before a Session record is considered old and ready to delete
-    OLD_IN_DAYS=7
+    OLD_IN_DAYS=Configuration.ood_bc_card_time
```

Comment:
I don't know if I like the pattern of keeping the constant but it actually being a variable. Can we make this a function instead? And I think the test cases will definitely start to fail then because they reference `OLD_IN_DAYS`.

```suggestion
    def old_in_days
      Configuration.ood_bc_card_time
    end
```

You can run tests yourself with `rake test` and `rake test TEST=test/models/batch_connect/session_test.rb`

Reply:
Thanks for the review! I'll make the changes.